### DISCUSSION TITLE
[PKGBUILD] Software with complicated licenses

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MINGW-packages#19885] [General]
Take this for example: https://github.com/modula3/cm3

I have no idea what to write in `license`.

**Comment by MehdiChinoune** [comment]
I see it https://github.com/modula3/cm3?tab=License-1-ov-file as https://spdx.org/licenses/BSD-4-Clause-UC.html

**Reply by ghost** [reply]
Please have a look at the various `COPYRIGHT` files.

**Reply by MehdiChinoune** [reply]
but the official one is the mentioned above
